docs/commands/bootstrap/README.md

```diff
@@ -30,25 +30,6 @@ $ gitops bootstrap
   [--commit-status-tracker]
 ```
 
-| Flag                                  | Description |
-| ------------------------------------- | ----------- |
-| --dockercfgjson                       | Filepath to config.json which authenticates the image push to the desired image registry. |
-| --gitops-repo-url                     | Provide the URL for your GitOps repository e.g. https://github.com/organisation/repository.git |
-| --gitops-webhook-secret               | Optional. Provide a secret that we can use to authenticate incoming hooks from your Git hosting service for the GitOps repository. (if not provided, it will be auto-generated)|
-| --help                                | Help for bootstrap flags. |
-| --image-repo                          | Image repository of the form <registry>/<username>/<repository> or <project>/<app> which is used to push newly built images. |
-| --image-repo-internal-registry-hostname          | Host-name for internal image registry e.g. docker-registry.default.svc.cluster.local:5000, used if you are pushing your images to the internal image registry |
-| --output                              | Path to write GitOps resources (default ".") |
-| --prefix                              | Add a prefix to the environment names(Dev, stage,prod,cicd etc.) to distinguish and identify individual environments. |
-| --service-repo-url                    | Provide the URL for your Sevice repository e.g. https://github.com/organisation/repository.git which is source code to your first application. |
-| --service-webhook-secret              | Optional. Provide a secret that we can use to authenticate incoming hooks from your Git hosting service for the GitOps repository. (if not provided, it will be auto-generated)|
-| --overwrite                           | Optional. Overwrites previously existing GitOps configuration (if any) (default false) |
-| --sealed-secrets-ns string            | Optional. Namespace in which the Sealed Secrets operator is installed, automatically generated secrets are encrypted with this operator (default "cicd") |
-| --sealed-secrets-svc string           | Optional. Name of the Sealed Secrets Services that encrypts secrets (default "sealedsecretcontroller-sealed-secrets"") |
-| --git-host-access-token string  | Optional. Used to authenticate repository clones, and commit-status notifications (if enabled)|
-| --private-repo-driver string          | Optional. If your Enterprise Git repositories are on a custom domain, please indicate which driver to use github or gitlab|
-| --commit-status-tracker bool          | Optional. Enable or disable the commit-status-tracker which reports the success/failure of your pipelineruns to GitHub/GitLab|
-
```

Comment:
I'd probably keep flag descriptions until we have a way to automatically generate docs for them.